### PR TITLE
Make the README prettier

### PR DESCRIPTION
**Pull request by noodlesatgithub**

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
So … basically centering headlines and inserting the mascot?

File: README.md [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
+<p align="center">Works with HTML, Markdown, Liquid, Nunjucks, Handlebars, Mustache, EJS, Haml, Pug, and JavaScript Template Literals.</p>
 
+<h3 align="center"><a href="https://github/11ty">GitHub</a> · <a href="https://twitter.com/eleven_ty">Twitter</a> · <a href="https://11ty.dev/docs/">Docs</a> · <a href="https://opencollective.com/11ty">Support us on Open Collective</a> · <a href="https://github.com/11ty/eleventy/stargazers">Star Eleventy on GitHub</a> · <a href="https://discord.com/invite/GBkBy9u">Discord</a> · <a href="https://npmjs.com/org/11ty">npm</a> · <a href="https://github.com/11ty/eleventy/actions?query=workflow%3A.github%2Fworkflows%2Fci.yml">GitHub Actions CI</a>
```
Comment (pdehaan) [inline README.md:9]: FWIW, this `<h3>` doesnt seem to be closed anywhere.

**Comment by noodlesatgithub** [comment]
Basically, but it does make the README look nicer (in my opinion).

**Comment by binyamin** [comment]
Just an FYI, both the logo and the mascot look rather big since there's so little text. I'd be happy to make a single banner to use instead.

**Comment by noodlesatgithub** [comment]
Just fixed it :)

**Comment by zachleat** [comment]
Ah—really sorry to close this but I’d like to keep the README pretty simple and markdown-ish. Really it’s just a placeholder to punt people to our website.

**Pull request closed without merging**